src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -411,6 +456,46 @@ public void AddHostObject(string name, FormulaType type, Func<IServiceProvider,
                 displayName: default);
 
             _variables.Add(hostDName, info);
+        }
+
+        internal void AddType(DName typeName, FormulaType type)

```

Comment:
>AddType [](http://example.com/codeflow?start=22&length=7)

Can this be public? This seems extremely useful. 
(The other helpers would be fine too)